src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx

```diff
@@ -416,27 +417,31 @@ export default function QueryEditorTopRow(props: QueryEditorTopRowProps) {
   );
 
   const datasetSelector = <>{renderDatasetSelector()}</>;
-
   return (
-    <EuiFlexGroup
-      className={classes}
-      responsive={!!props.showDatePicker}
-      gutterSize="xs"
-      direction="column"
-      justifyContent="flexEnd"
-    >
+    <>
+      {/* Portals for dataset selector and date picker */}
```

Comment:
can we remove this comment? Its pretty clear what goes here, and if we add more stuff here later, i'm pretty sure its going to get stale real fast